### ISSUE TITLE
Team Check should work for services and employees equally

### ISSUE THREAD

**Issue by mo-gr** [zalando/innkeeper#209]
The team-service does not return a team for service-tokens. For that, the application registry (kio) must be queried to obtain a team name:

```
curl -v https://kio/apps/app-name -H "Authorization: Bearer $token" | jq .team_id
```

**Comment by mo-gr** [comment]
Team check should also be cached for a reasonable amount of time (1h?). No need to put high pressure on the team service during bulk updates

**Comment by mo-gr** [comment]
The caching was solved by #214
